.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a report to help us improve pyright
 title: ''
-labels: ''
+labels: bug
```

Comment:
I think that @erictraut typically avoids labelling issues as bugs until they've been looked at, hence why this is set to a blank. (The other change seems fine given the current state of the labels.)

Reply:
Related: https://github.com/microsoft/pyright/pull/2155#discussion_r683814633